andela/trime
# Show toast on organization / project deletion

### What does this PR do?
Adds a toast to be shown when a user deletes a project to indicate success or failure.

### How should this be manually tested?
Go to the dashboard and delete an organization (you can create another organization if you don't want to deleting an existing one). You should see a success toast when the organization is deleted successfully or an error toast if deletion fails.

Go to an organization page and delete a project (you can create another project if you don't want to deleting an existing one). You should see a success toast when the project is deleted successfully or an error toast if deletion fails.

### What are the relevant pivotal tracker stories?
[After deleting  a project or an org, user should see a toast message indicating success or failure](https://www.pivotaltracker.com/story/show/128049257)
